### PR TITLE
OSDOCS-10279: adds 4.15.13 relnotes Microshift

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.15

Issue:
[OSDOCS-10279](https://issues.redhat.com/browse/OSDOCS-10279)

Link to docs preview:
[4.15.13 async release note](https://75829--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-15-release-notes.html#microshift-4-15-13-dp)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by bhardesty — COMMENTED** [review]
LGTM!

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 1]
```diff
@@ -287,4 +287,20 @@ Issued: 2024-05-09
...
+
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[discrete]
+[id="microshift-4-15-13-enhancements"]
+==== Enhancements
+
+GitOps with Argo CD documentation is now available::
+* With this release, documentation for using GitOps is now available. Using GitOps means you can update a single Git repository and automate the deployment of new applications or updates to existing ones. For more information, see xref:../microshift_install/microshift-install-rpm.adoc#microshift-installing-rpms-for-gitops_microshift-install-rpm[Installing the GitOps Argo CD manifests from an RPM package](link:https://issues.redhat.com/browse/OSDOCS-9729[OSDOCS-9729]) and xref:../microshift_running_apps/microshift-gitops.adoc#microshift-gitops[Automating application management with the GitOps controller](link:https://issues.redhat.com/browse/OSDOCS-9764[OSDOCS-9764])
```
Comment (ShaunaDiaz) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:306]:
```suggestion
* With this release, documentation for using GitOps is now available. Using GitOps means you can update a single Git repository and automate the deployment of new applications or updates to existing ones. For more information, see xref:../microshift_install/microshift-install-rpm.adoc#microshift-installing-rpms-for-gitops_microshift-install-rpm[Installing the GitOps Argo CD manifests from an RPM package] and xref:../microshift_running_apps/microshift-gitops.adoc#microshift-gitops[Automating application management with the GitOps controller].
```

Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:306]: This is not a bug fix, so we do not use the Jira issue here.

Comment (aravipra) [reply microshift_release_notes/microshift-4-15-release-notes.adoc]: okay

**Pull request merged**